exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -681,35 +674,29 @@ func (e *Exchange) SubmitOrder(ctx context.Context, s *order.Submit) (*order.Sub
 	if err != nil {
 		return nil, err
 	}
+	// TODO: These are just here to allow a minimal POC on just the smart request type
```

Comment:
🚧 Change request:
😳  The example POC I supplied was just an example Proof of Concept.
I expected all the other types to be changed, and for these `*String` methods to be removed.

Reply:
Ohh my bad

Reply:
Erm... You removed the comment. But didn't actually action this.